src/serac/numerics/odes.cpp

```diff
@@ -171,9 +171,12 @@ void SecondOrderODE::Solve(const double time, const double c0, const double c1,
   bool implicit = (c0 != 0.0 || c1 != 0.0);
   if (implicit) {
     if (enforcement_method_ == DirichletEnforcementMethod::DirectControl) {
-      d2U_dt2_ = (U_ - u) / c0;
-      dU_dt_   = du_dt;
-      U_       = u;
+      // TODO: MFEM PR#3064 explicitly deleted the const vector operator-.  This
+      // is in active discusion and may be un-deleted. Original line commented.
+      // d2U_dt2_ = (U_ - u) / c0;
```

Comment:
Confirmed these can revert back to the commented code after the mfem pr goes through.